sdk/resourcemanager/Azure.ResourceManager/src/autorest.md

```diff
@@ -828,6 +836,10 @@ directive:
     where: $.definitions.ManagementGroupProperties.properties.tenantId
     transform: >
       $['format'] = "uuid"
+  - from: management.json
+    where: $.paths
+    transform: >
+      $['/providers/Microsoft.Management/managementGroups/{groupId}'].delete['x-ms-long-running-operation-options']['final-state-schema'] = undefined;
```

Comment:
if we have to do this, does this mean there is still something wrong in the corresponding tsp?
We need to figure this out because there is no way of doing the same when we generate from tsp.